src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -597,13 +596,13 @@ public void onDestroyActionMode(ActionMode mode) {
   public void handleEvent(@NonNull DcEvent event) {
     if (event.getId() == DcContext.DC_EVENT_CONNECTIVITY_CHANGED) {
       Activity activity = getActivity();
-      if (activity != null) {
+      if (activity instanceof ConversationListActivity) {
         ((ConversationListActivity) activity).refreshTitle();
       }
 
     } else if (event.getId() == DcContext.DC_EVENT_SELFAVATAR_CHANGED) {
       Activity activity = getActivity();
-      if (activity != null) {
+      if (activity instanceof ConversationListActivity) {
```

Comment:
I fixed a small crash here: sometimes `activity` is a ConversationListArchiveActivity, not a ConversationListActivity.